Added: docu of test and results

mattibf opened the pull request.

mattibf committed:
Added: docu of test and results

mattibf committed:
Added: kafka_python_client_perf: Analysis of throughput of different kafka Python clients

witekest commented on kafka_python_client_perf/monascaInvestigationKafkaPythonAPIs.md:
```diff
@@ -0,0 +1,694 @@
+Example: Confluent client
+-------------------------
+
+-   **createProducer**:  
```
Code snippets are incorrectly formatted. I would remove them from here completely.

mattibf committed:
Merge branch 'kafka_client_perf' of https://github.com/monasca/monasca-perf into kafka_client_perf

mattibf committed:
Throughput test for kafka python clients:
Changes since last push:
Throughput tests were executed again.
1. monascaInv...md, media/image1-3.png: Reflect new test results
2. kafka-py-client-test.py:
- Some test cases added
- Simplify code
